config.yml

```diff
@@ -0,0 +1,99 @@
+defaults:
+  app:
+    publishMetaData: false
```

Comment:
`!env:bool PUBLISH_METADATA` is what I would use...

Maybe compare to: https://github.com/taskcluster/taskcluster-queue/blob/master/config.yml